style_guide.md

```diff
@@ -339,11 +339,11 @@ $ adb pull <remote_file> <target_destination>
 When they do not occur in a code block, place the following code-related keywords in backticks (``), double straight quote marks (""), or leave unpunctuated according to the table:
 
 |    Backticks    | Quotation Marks | No Punctuation |
-| --------------- | --------------- | -------------- |
+| --- | --- | --- |
 | function names  |  section titles | application name |
 |   method names  |  chapter titles | folder names  |
-|    commands     |    book titles  | navigation (e.g., Home -> Menu)|
-|   class names   | flags values (e.g., "true", lowercase) | ID #s |
+|    commands     |    book titles  | ID #s |
```

Comment:
ID #s was already there but I cannot tell what it is. We should make it clear

Reply:
So the IDs came in 2 years ago... I totally forgot about it... maybe we should remove it?